Documentation/ExtensionArchitecture/HowTo/Configuration.rst

```diff
@@ -0,0 +1,100 @@
+.. include:: /Includes.rst.txt
+.. index:: Extension; Configuration
+.. _extension_configuration:
+
+=============
+Configuration
+=============
+
+There are several possibilities to make your extension configurable. From the
+various options described here each differs in:
+
+*   the scope to what the configuration applies (extension, pages,
+    plugin)
+*   the access level required to make the change (editor, admin)
+
+.. index::
+   TypoScript
+   Configuration; TypoScript
+
+TypoScript and constants
+========================
+
+You can define configuration options using TypoScript.
+These options can be changed via TypoScript constants and setup in the backend.
+The changes apply to the current page and all subpages.
+
+.. seealso::
+
+   * :ref:`extbase_typoscript_configuration`
```

Comment:
```suggestion
   * :ref:`Extbase TypoScript configuration <extbase_typoscript_configuration>`
```
I expected a generic approach when clicking on it.

Reply:
How would you rather phrase them?

Reply:
That's why I suggested changing the link title. Without the change it is just "TypoScript configuration" (where I would expect the generic approach).